server/executor/trace_poller.go

```diff
@@ -75,6 +75,15 @@ type PollingRequest struct {
 	count int
 }
 
+func NewPollingRequest(ctx context.Context, test model.Test, run model.Run, count int) *PollingRequest {
```

Comment:
Can't you use this on the production code as well? Now It's only used for tests at the moment.

Reply:
Makes sense, adding it!